### PR TITLE
Updates for additional UI kit options

### PR DESCRIPTION
**Pull request by JoshuaPartlow**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/design-teams-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c42ff65e122672301bef7aafa696e99acfd6d67f/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-overview.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b96208ef-879b-4bf8-921b-de8f6a3a3d03) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-teams-app-ui-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c42ff65e122672301bef7aafa696e99acfd6d67f/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-ui-templates.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9ba64204-a62c-46de-b001-2b8a55cbd4e4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c42ff65e122672301bef7aafa696e99acfd6d67f/msteams-platform%2Fm365-apps%2Foverview.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a4b7e527-aa20-4b9e-b317-7b6cb7c34300) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by erikadoyle — APPROVED** [review]
Thanks @JoshuaPartlow, LGTM. @surbhigupta12, I'll provide you more context around this change at our next sync.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/design-teams-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9a4e96d00181a0a855877957b227b071c2d7aefa/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-overview.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0346078c-435d-4448-bbcb-6bb888fb5921) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-teams-app-ui-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9a4e96d00181a0a855877957b227b071c2d7aefa/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-ui-templates.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c44a2204-4021-4b86-98dc-ad1d1b401cca) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9a4e96d00181a0a855877957b227b071c2d7aefa/msteams-platform%2Fm365-apps%2Foverview.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a2f57be6-c052-49ad-a3ab-526a18f61b90) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

File: msteams-platform/concepts/design/design-teams-app-overview.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ Understand how people add, use, and manage Teams apps to make the most of each c
 
 ## UI templates
 
-Quickly create complex, high-fidelity designs with [templates for common Teams use cases and workflows](design-teams-app-ui-templates.md).
+Quickly create complex, high-fidelity designs with [templates for common Teams use cases and workflows](design-teams-app-ui-templates.md). For [apps extended across Microsoft 365](../../m365-apps/overview.md#app-planning-and-design), app playbooks and UI templates are also available.
```
Comment (v-sdhakshina) [inline msteams-platform/concepts/design/design-teams-app-overview.md:127]:
```suggestion
Quickly create complex, high-fidelity designs with [templates for common Teams use cases and workflows](design-teams-app-ui-templates.md). For [apps extended across Microsoft 365](../../m365-apps/overview.md#app-planning-and-design), app playbooks and UI templates are also available.
```

**Review by v-sdhakshina — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/design-teams-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f13c582673ed92eba4268155736c02b1a7a40497/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-overview.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8e093ecb-714a-4107-b2a6-387414c5bd49) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-teams-app-ui-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f13c582673ed92eba4268155736c02b1a7a40497/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-ui-templates.md) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/13e9efa5-a1f8-4325-9d5a-bf66615c6cfa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/design-teams-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0f162f7074ba686219f58d29cf9c8089a72b0c21/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-overview.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7cc4dfbc-e532-4aab-9506-572191fa47c2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-teams-app-ui-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0f162f7074ba686219f58d29cf9c8089a72b0c21/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-ui-templates.md) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7de04019-eece-44a8-8fc2-92c7936883d5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0f162f7074ba686219f58d29cf9c8089a72b0c21/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a7c89310-53ea-4193-989e-9b1647cfc496) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/design-teams-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/966d4913082099800ccc36156bcc747c94e435d8/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-overview.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e899286d-327b-4e01-b27c-29e599ced020) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-teams-app-ui-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/966d4913082099800ccc36156bcc747c94e435d8/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-ui-templates.md) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/925076c0-fac2-43ca-b81d-49951dbfa1ab) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/966d4913082099800ccc36156bcc747c94e435d8/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c206c40e-4efa-4f26-88a1-b9169cc2c2e0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/design-teams-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/04c7f830c314de9a9752cb71b0d539ce0946f178/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-overview.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/58dbb7b4-21df-41f6-a56f-f622252d63ae) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-teams-app-ui-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/04c7f830c314de9a9752cb71b0d539ce0946f178/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-ui-templates.md) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/38d320ae-55c3-44ca-a711-a4d03bf057ed) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/04c7f830c314de9a9752cb71b0d539ce0946f178/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6037070d-323c-4e2e-84f7-e32b0ff19e82) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/design-teams-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bec4ee0c918b8e93fad8e182d5cd9a18b8fe5cc7/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-overview.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eafedd02-f157-4cd0-abcb-22aae828bdbd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/design/design-teams-app-ui-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bec4ee0c918b8e93fad8e182d5cd9a18b8fe5cc7/msteams-platform%2Fconcepts%2Fdesign%2Fdesign-teams-app-ui-templates.md) | 98 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/93e36d33-d7a2-4610-bfcd-f26054adfd8e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bec4ee0c918b8e93fad8e182d5cd9a18b8fe5cc7/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50341a4c-2875-48c6-8f94-50a6a9cd4f76) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-sdhakshina — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**